app/services/DataStoreService.scala

```diff
@@ -37,9 +38,8 @@ class DataStoreService @Inject() (httpClient: HttpClientV2, metricsReporter: Met
 
   val log: Logger = Logger(this.getClass)
 
-  def getAllEoriHistory(eori: EORI)(implicit hc: HeaderCarrier): Future[Seq[EoriHistory]] = {
-    val dataStoreEndpoint = s"${appConfig.customsDataStore}/eori/$eori/eori-history"
-    val emptyEoriHistory  = Seq(EoriHistory(eori, None, None))
+  def getAllEoriHistory()(implicit hc: HeaderCarrier): Future[Seq[EoriHistory]] = {
+    val dataStoreEndpoint = s"${appConfig.customsDataStoreWithEori}/eori-history"
```

Comment:
Could you please make use of `appConfig.customsDataStore `instead of `appConfig.customsDataStoreWithEori` as the logical distinction for the route is like this, it has two parts one comes from `appConfig.customsDataStore` and second is `/eori/eori-history.` To split /eori from the second part is not logical distinction. 

After making the change it should be like this

val dataStoreEndpoint = s"${appConfig.customsDataStore}/eori/eori-history"

Above comment is applicable for other methods(of this class) as well.